### ISSUE TITLE
Consider paginating delinquent students view

### ISSUE THREAD

**Issue by theo-o** [tjcsl/ion#756]
Currently, if a large range is selected, the action takes a long time and can return a lot of data.

Care should be taken to not break the sorting functionality.

**Comment by alanzhu0** [comment]
I don't think this is too important as absences are cleared each year so the amount of data is not too overwhelming.

**Comment by sinanchung** [comment]
Could this issue be assigned to me?

**Comment by aarushtools** [comment]
@sinanchung take a look at #1907
